notes2.md

```diff
@@ -0,0 +1,49 @@
+### Note about Setting Up Test Doubles
```

Comment:
Great idea to capture thoughts or implementation ideas for later and keep them with the code 👍